src/nidcpower/examples/nidcpower_source_delay_measure.py

```diff
@@ -22,16 +23,16 @@ def example(resource_name, channels, options, voltage1, voltage2, delay):
         session.current_limit = .06
         session.voltage_level_range = 5.0
         session.current_limit_range = .06
-        session.source_delay = delay
+        session.source_delay = datetime.timedelta(seconds=delay)
```

Comment:
Instead of declaring timeout as a float on line 16, declare it as a timedelta down here. Will clean up the code later.
Alternatively, you could rely on default timeout, which is 10 s. If the user of the example uses a source delay longer than 10 s (why would they do that!!!!) then they get a timeout error. Cleaner code in my opinion.

Reply:
The default timeout for `fetch_multiple()` is one second: `fetch_multiple(self, count, timeout=datetime.timedelta(seconds=1.0))`

Changed to declare `timeout` as `datetime.timdelta`